api/api_ethereum.go

```diff
@@ -1396,11 +1396,15 @@ func (api *EthereumAPI) Accounts() []common.Address {
 // rpcMarshalHeader marshal block header as Ethereum compatible format.
 // It returns error when fetching Author which is block proposer is failed.
 func (api *EthereumAPI) rpcMarshalHeader(head *types.Header) (map[string]interface{}, error) {
-	proposer, err := api.publicKlayAPI.b.Engine().Author(head)
-	if err != nil {
-		// miner is the field Klaytn should provide the correct value. It's not the field dummy value is allowed.
-		logger.Error("Failed to fetch author during marshaling header", "err", err.Error())
-		return nil, err
+	var proposer common.Address
+	var err error
+	if head.Number.BitLen() != 0 {
```

Comment:
If you want to check the head.Number has not zero value, It seems to be appropriate to use Sign() of big.Int type 
Check https://pkg.go.dev/math/big#Int.Sign

Reply:
@sirano11 
Thanks. I updated it.

I checked the what would be good option for this and below is the result of Benchmark test.

Comparing using `Sign` is more faster than using BitLen.

```
goos: darwin
goarch: amd64
pkg: play
cpu: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
BenchmarkBitLen
BenchmarkBitLen-16      581610451                1.940 ns/op
BenchmarkSign
BenchmarkSign-16        942986775                1.225 ns/op
PASS
ok      play    2.881s
```  

test source code: [big_bench_test.go](https://gist.github.com/aeharvlee/a90a094667e43454b04729d68f337473)